### PR TITLE
Customize ERROR_INVALID_WINDOW_HANDLE exception text to make window interop helpers discoverable

### PR DESCRIPTION
**Pull request by Scottj1s**

### CODE REVIEW

File: src/WinRT.Runtime/ExceptionHelpers.cs [hunk 1]
```diff
@@ -154,6 +155,13 @@ private static Exception GetExceptionForHR(int hr, bool useGlobalErrorState, out
                     case E_ELEMENTNOTENABLED:
                         ex = new Microsoft.UI.Xaml.Automation.ElementNotEnabledException();
                         break;
+                    case ERROR_INVALID_WINDOW_HANDLE:
+                        ex = new System.Runtime.InteropServices.COMException(
+@"Invalid window handle. (0x80070578)
+Consider IWindowNative, IInitializeWithWindow
+See also https://aka.ms/cswinrt/interop#windows-sdk",
```
Comment (angelazhangmsft) [inline src/WinRT.Runtime/ExceptionHelpers.cs:162]: consider linking to Microsoft Docs instead? https://docs.microsoft.com/windows/apps/desktop/modernize/winrt-com-interop-csharp

Comment (Scottj1s) [reply src/WinRT.Runtime/ExceptionHelpers.cs:162]: added mdocs link to interop md

File: docs/interop.md [hunk 2]
```diff
@@ -95,7 +95,28 @@ SetString(MarshalString.GetAbi(marshalStr));
...
+var folderPicker = new Windows.Storage.Pickers.FolderPicker();
+folderPicker.FileTypeFilter.Add("*");
+WinRT.Interop.InitializeWithWindow.Initialize(folderPicker, hwnd);
+var folder = await folderPicker.PickSingleFolderAsync();
+```
+
+The Windows SDK projection also provides wrappers for all WinRT interop interfaces included in the Universal API Contract, such as ***Windows.Security.Credentials.UI.UserConsentVerifierInterop***
+
+For more info, see:
+[Call WinRT COM interop interfaces from .NET 5+ apps](https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/winrt-com-interop-csharp)
```
Comment (angelazhangmsft) [inline docs/interop.md:119]: remove en-us: https://docs.microsoft.com/windows/apps/desktop/modernize/winrt-com-interop-csharp

**Review by angelazhangmsft — APPROVED** [review]

**Comment by j0shuams** [comment]
Awesome 👍

File: docs/interop.md (hunk 2 above)
Comment (Scottj1s) [reply docs/interop.md:119]: oops missed your comment - will pick this up later

**Pull request merged**